frontend/pages/shopping-lists/index.vue

```diff
@@ -21,6 +21,12 @@
       <BaseButton create @click="createDialog = true" />
     </v-container>
 
+    <v-container v-if="shoppingListChoices.length === 0">
```

Comment:
This is a nitpick, no need to change, but for the sake of learning FYI this also works:
```vue
<v-container v-if="!shoppingListChoices.length">
```

It's preference, but this way is more typical

Reply:
No, that's great feedback, I will change it. I'm new to Vue so every bit of "for the sake of learning" is helpful, thank you.